server/lib/chart.js

```diff
@@ -0,0 +1,127 @@
+const _ = require('lodash');
+const util = require('./util');
+
+module.exports = {
+  barChar,
+};
+
+/**
+ * @function barChar
+ * This function help to skip some manipulation for respecting chartjs lib format...
+ */
+
+function barChar(params) {
+  const {
+    label, data, item, yAxesLabelString,
+    xAxesLabelString, canvasId,
+  } = params;
+
+  const chart = {
+    labels : [],
+    data : [],
+  };
+  const serieMap = {};
+
+  const dailyData = _.groupBy(data, label);
```

Comment:
Could you add in the description of the function, the format of different variables (data, label, item, etc.) so that we could easily reuse it